### PR TITLE
[hi] Localize  content/hi/docs/reference/glossary/job.md

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: Amol Mote <amolmote201@gmail.com>

Fixes #35600 [hi] Localize content/hi/docs/reference/glossary/job.md

### CODE REVIEW

File: content/hi/docs/reference/glossary/job.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक परिमित या बैच कार्य जो पूरा होने तक चलता है।
+
+<!--more--> 
+
+जॉब एक या अधिक {{<glossary_tooltip text="पॉड" term_id="pod" >}} ऑब्जेक्ट बनाता है और सुनिश्चित करता है कि उनमें से एक निर्दिष्ट संख्या सफलतापूर्वक समाप्त हो जाए। जैसी ही पॉड्स सफलतापूर्वक पूर्ण होते हैं, जॉब उस सफल समापन को ट्रैक करता है।
```
Comment (kundan2707) [inline content/hi/docs/reference/glossary/job.md:19]: जैसे ही पॉड्स

**Comment by Babapool** [comment]
@amolmote can please push an empty commit once again to retrigger the check so we can make this PR ready to merge. Thanks

**Comment by anubha-v-ardhan** [comment]
@Babapool I believe, we can use prow to retest failed checks.

/retest

**Comment by bishal7679** [comment]
@anubha-v-ardhan I didn't get the reason why its throwing buiding error!
Same happened with my one PR and the checks will keep failing inspite of pushing an empty commit or retest
Should we refer this followed by a new PR by taking ref from this??
As @amolmote seems to be inactive quite a while
WDYT?

**Comment by anubha-v-ardhan** [comment]
@bishal7679 We can discuss with SIG-Docs to about the issue on some of the PRs. Replacing this PR with a new one can be a quick solution but let's wait for some time (say a week) if @amolmote is still active with this PR.

**Review by kundan2707 — COMMENTED** [review]
/lgtm

**Review by Babapool — COMMENTED** [review]
Thanks!
/lgtm

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Pull request merged**